### PR TITLE
add welcome

### PR DESCRIPTION
**Pull request by iiiiii1wepfj**

### CODE REVIEW

File: plugins/welcome.py [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+    db.commit()
+
+
+def disable_welcome(chat_id):
+    dbc.execute('UPDATE groups SET welcome_enabled = ? WHERE chat_id = ?', (False, chat_id))
+    db.commit()
+
+
+@Client.on_message(filters.command("setwelcome", prefix) & filters.group)
+@require_admin()
```
Comment (AndrielFR) [inline plugins/welcome.py:41]: "can_change_info" permission required.

File: plugins/welcome.py [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+async def reset_welcome_message(c: Client, m: Message):
+    set_welcome(m.chat.id, None)
+    await m.reply_text(f"The welcome message for {m.chat.title} were setted again to the default welcome message of the bot.")
+
+
+@Client.on_message(filters.new_chat_members & filters.group)
+async def greet_new_members(c: Client, m: Message):
+    chat_title = m.chat.title
+    chat_id = m.chat.id
+    first_name = m.from_user.first_name
```
Comment (AndrielFR) [inline plugins/welcome.py:81]: This is wrong, you have to use a `for loop`.
```py
for user in m.new_chat_members:
   ...
```

File: plugins/welcome.py (hunk 1 above)
Comment (iiiiii1wepfj) [reply plugins/welcome.py:41]: no, not need, need and this permission not make this cant to be used on translation groups

File: plugins/welcome.py (hunk 2 above)
Comment (iiiiii1wepfj) [reply plugins/welcome.py:81]: working good in my testing group

Comment (alissonlauffer) [reply plugins/welcome.py:81]: > working good in my testing bot

It works, but will only greet the first member when someone add more than one user per time.

Comment (iiiiii1wepfj) [reply plugins/welcome.py:81]: you can try

**Pull request merged**